src/main/scala/uk/gov/hmrc/ct/accounts/CompanyAddress.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.accounts
+
+import uk.gov.hmrc.ct.accounts.formats.CompanyAddressFormatter
+import uk.gov.hmrc.ct.box.{CtBoxIdentifier, CtValue, Input}
+
+case class CompanyAddress(addressLine1: Option[String] = None,
+                          addressLine2: Option[String] = None,
+                          addressLine3: Option[String] = None,
+                          addressLine4: Option[String] = None,
+                          city: Option[String] = None,
+                          county: Option[String] = None,
+                          postCode: Option[String] = None,
+                          country: Option[String] = None)
+  extends CtBoxIdentifier(name = "Registered Company Address")
+  with CtValue[CompanyAddress]
+  with Input {
+
+  override def value = this
+
+  override def asBoxString = CompanyAddressFormatter.asBoxString(this)
+
+}
```

Comment:
Box numbers are just a CATO invention. In case of this "box" it's not an input field. Value for it is loaded from COTAX or CoHo. When printed there's no box number next to that address. Knowing that you still want to "make up" a box number for it?